File: 2012/fall/hacker2/generator.c
#define _XOPEN_SOURCE
#include <stdio.h>
#include <unistd.h>

int main(void)
{
    printf("caesar:%s\n", crypt("13", "50"));
    printf("cs50:%s\n", crypt("12345678", "50"));
    printf("jharvard:%s\n", crypt("crimson", "50"));
    printf("malan:%s\n", crypt("crimson", "HA"));
    printf("nate:%s\n", crypt("bacon", "50"));
    printf("rbowden:%s\n", crypt("password", "50"));
    printf("skroob:%s\n", crypt("12345", "50"));
    printf("tmacwilliam:%s\n", crypt("n00b", "50"));
    printf("zamyla:%s\n", crypt("l33t", "50"));
    return 0;
}

File: 2012/fall/pset7/generator.php
#!/bin/env php
<?php

    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "caesar", crypt("13", '$1$50$'));
    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "cs50", crypt("12345678", '$1$50$'));
    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "jharvard", crypt("crimson", '$1$50$'));
    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "malan", crypt("crimson", '$1$HA$'));
    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "nate", crypt("bacon", '$1$50$'));
    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "rbowden", crypt("password", '$1$50$'));
    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "skroob", crypt("12345", '$1$50$'));
    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "tmacwilliam", crypt("n00b", '$1$50$'));
    printf("INSERT INTO users (username, hash) VALUES('%s', '%s');\n", "zamyla", crypt("l33t", '$1$50$'));

?>
